rgb2hex.js

```diff
@@ -14,17 +14,17 @@
         }
 
         // parse input
-        var digits = /(.*?)rgb(a)*\((\d+),(\d+),(\d+)(,[0-9]*\.*[0-9]+)*\)/.exec(color.replace(/\s+/g,''));
+        var digits = /^rgba?\((\d+),(\d+),(\d+)(,[0-1]?\.?[0-9]+)?\)$/.exec(color.replace(/\s+/g,''));
```

Comment:
I wonder if `color.replace(/\s+/gm,'')` would be safer? Anticipate any use cases where a multiline string is passed in, e.g. `'rgb(1,2,3)\n'`?

...or maybe I should just remove the terminating `$`, since maybe someone will pass in something like `rgb(1,2,3);`? (Is that even realistic/to be considered valid?)

Please let me know what's referred, in which case I'll also add a couple more tests.

Reply:
> wonder if color.replace(/\s+/gm,'') would be safer? Anticipate any use cases where a multiline string is passed in, e.g. 'rgb(1,2,3)\n'?

I think such cases are possible as well `rgb(1,2,3);` .. everything else e.g. that has more than one character should be considered invalid